### PR TITLE
P0 images & input components

### PR DESCRIPTION
**Pull request by Naveenchandar**

### CODE REVIEW

File: image.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    background-color: #F5F7F9;
+    min-height: 100px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 1rem;
+    margin-bottom: 1.5rem;
+}
+
+.img {
```
Comment (rohanmathur91) [inline image.css]:
```suggestion
You can follow any one naming convention, either .img_somename or .image_somename for consistency.
```

**Review by rohanmathur91 — APPROVED** [review]
Rest looks good to me.

File: image.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.image_group {
+    background-color: #F5F7F9;
+    min-height: 100px;
```
Comment (prakashsakari) [inline image.css]: You can use rem instead of pixels over here.

Comment (Naveenchandar) [reply image.css]: Done

File: image.css (hunk 1 above)
Comment (Naveenchandar) [reply image.css]: Done

**Review by maitrakhatri — APPROVED** [review]
looks good to me

**Pull request merged**